provision/core/mailhog/provision.sh

```diff
@@ -10,13 +10,22 @@ function mailhog_setup() {
 
   if [[ ! -e /usr/local/bin/mailhog ]]; then
     vvv_info " * Installing MailHog"
-    curl --silent -L -o /usr/local/bin/mailhog https://github.com/mailhog/MailHog/releases/download/v1.0.0/MailHog_linux_amd64
+
+    if [[ "aarch64" == $(uname -m) ]]; then
+      curl --silent -L -o /usr/local/bin/mailhog https://github.com/evertiro/MailHog/releases/download/v1.0.0-M1/MailHog_linux_arm64
+    else
+      curl --silent -L -o /usr/local/bin/mailhog https://github.com/mailhog/MailHog/releases/download/v1.0.0/MailHog_linux_amd64
+    fi
```

Comment:
this could be separated into its own PR, Ubuntu 18 runs on Arm too ( e.g. [Raspberry Pis](https://maker.pro/raspberry-pi/projects/install-ubuntu-18044-lts-on-your-raspberry-pi-board)  )

Reply:
Agreed

Reply:
This is resolved with #2423